src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -739,67 +848,79 @@ export async function deposit(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: DepositRequest
-): Promise<string> {
+): Promise<DepositResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const asset = await panopticish.deposit(
     wallet,
     req.collateralTracker,
     BigNumber.from(req.assets)
   );
-  return asset;
+  return {
+    shares: asset.shares
```

Comment:
Maybe rename to `sharesReceived`? What do you think?

Reply:
That's fine, was just matching the property name on the ABI. Changed: 462b3018d99060ac4040287b7f7b5eac94c23565